Formula/mas.rb

```diff
@@ -33,7 +33,7 @@ class Mas < Formula
 
   def install
     system "script/build"
-    system "script/install", prefix
+    bin.install ".build/release/mas"
```

Comment:
Note that this path is different when you do a universal build:
https://github.com/mas-cli/mas/commit/bdcb99f236f7b286f87904afc9f99c620dbcae60

Reply:
`brew bump-formul-pr`, which my release process uses, doesn't build universal executables.